runall.py

```diff
@@ -97,6 +99,10 @@ def print_help():
     print("        Print out each test as it is executed")
     print("    -C, --color")
     print("        Enable color output")
+    print("    --email=<name>")
+    print("        If the results need to be emailed to someone, please provide the email name")
```

Comment:
Isn't this script out or our purview? When we merge back to the main VUnit branch is this change still applicable? At the very least we should take the @metrics.ca out of it.